Remotion/Web/Development.WebTesting/Configuration/WebTestConfigurationSection.cs

```diff
@@ -19,14 +19,15 @@
 using System.IO;
 using OpenQA.Selenium;
 using Remotion.Utilities;
+using Remotion.Web.Development.WebTesting.Configuration.Legacy;
 using Remotion.Web.Development.WebTesting.DownloadInfrastructure;
 
 namespace Remotion.Web.Development.WebTesting.Configuration
 {
   /// <summary>
   /// Loads the app.config and validates the parameter. Serves as DTO Object to transfer the app.config settings to the finer grained config classes.
   /// </summary>
-  public class WebTestConfigurationSection : ConfigurationSection
+  public class WebTestConfigurationSection : ConfigurationSection, IWebTestSettings
```

Comment:
Why is this type not moved to the Legacy section?

Reply:
This type is not moved as it would create a breaking change in all XML files that reference it via namespace.